src/ContentWithCircularProgress/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import clsx from 'clsx';
+
+import { CircularProgress, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  alignCenter: {
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+  },
```

Comment:
I think it's clearer to use "grid" method instead of "position: absolute" method to center an element.
Here is a [Codesandbox](https://codesandbox.io/s/center-circular-65c7x?file=/demo.js) to see the result

```suggestion
  alignCenter: {
    width: '100%',
    height: '100%',
    display: 'grid',
    justifyContent: 'center',
    alignItems: 'center',
  },
```